modules/javafx.graphics/.classpath

```diff
@@ -1,17 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="src/main/java"/>
-	<classpathentry kind="src" path="build/hlsl/Prism">
-		<attributes>
-			<attribute name="ignore_optional_problems" value="true"/>
-		</attributes>
-	</classpathentry>
 	<classpathentry kind="src" path="build/gensrc/jsl-prism"/>
-	<classpathentry kind="src" path="build/hlsl/Decora">
-		<attributes>
-			<attribute name="ignore_optional_problems" value="true"/>
-		</attributes>
-	</classpathentry>
 	<classpathentry kind="src" path="build/gensrc/jsl-decora"/>
```

Comment:
1. I will try to incorporate your changes, especially in apps/.  My initial goal was to fix the build of everything except apps/
2. I do get some warnings, they will be addressed by other fixes under JDK-8289379.  But others (ex: MacPasteboardShim) I do not see, as those have been fixed.  Could you please make sure you have no modified files and do a refresh?
3. Could you clarify your point about build/hlsl/Decora and build/hlsl/Prism please?  I noticed there is nothing there, and nothing seem to be put there.  Also, there are build/gensrc/jsl-prism and build/gensrc/jsl-decora directories being created, so I felt we can remove these entries from .classpath.  I do build on Mac though.

Reply:
1. The way the apps are structured is that each app is its own project, which means that there are many new projects to create and configure. It can be a bit of work, though I have done it in my branch already. If you want to leave it to round 2 of fixing the Eclipse files it's fine.

2+3. I worked directly from master and from your branch, so no files have been changed. The folders under `build` are OS-specific and are generated as needed by the build. Since you are on Mac, it would explain why the hlsl folders are empty, but they are not empty on Windows. This is why they are marked as "ignore_optional_problems" - to not produce errors even on the wrong system. Since they don't contain Java classes that are referenced by the build (as far as I could tell), it should be safe to remove the `build/...` source folders in general. I don't know if on Mac you get any issues, but I assume you don't.

Reply:
1. I would rather leave the apps/ for day 2, if I may.
2+3.  Currently, on Mac, I don't see any issues related to build/... folders.  I think, generally speaking, it's a bad idea to refer to derived folders as source - it complicates the build process in Eclipse.  Does my branch fail on windows in Eclipse, Nir?  I mean, do I need to make changes to address this comment?

Reply:
I don't get any errors on the graphics module, I'm suggesting that you remove the entries:
```
	<classpathentry kind="src" path="build/gensrc/jsl-prism"/>
	<classpathentry kind="src" path="build/gensrc/jsl-decora"/>
```

Reply:
good point.  removing

	<classpathentry kind="src" path="build/gensrc/jsl-prism"/>
	<classpathentry kind="src" path="build/gensrc/jsl-decora"/>